service/src/main/java/datawave/microservice/authorization/AuthorizationOperationsV2.java

```diff
@@ -31,25 +32,34 @@ public AuthorizationOperationsV2(JWTTokenHandler tokenHandler, CachedDatawaveUse
         super(tokenHandler, cachedDatawaveUserService, appCtx, busProperties);
     }
     
-    @ApiOperation(value = "Authorizes the calling user to produce a JWT value",
-                    notes = "The returned JWT can be passed to other calls in a header. For example: \"Authorization: bearer <JWT value>\".\n"
-                                    + "The user can be determined with from the supplied client certificate or trusted headers ("
-                                    + "X-SSL-clientcert-subject/X-SSL-clientcert-issuer).")
+    // remove final caller if there are any proxied users
```

Comment:
Similar comment to the one above.  Lets make the name of this method a little more obvious as to what it is doing.

Reply:
Documented what the method is doing a little more in the method comments